backend/app/models/residences.py

```diff
@@ -0,0 +1,39 @@
+from . import db
+import enum
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+#- id
+#- date joined 
+#- status: 
+
+class StatusEnum(enum.Enum):
+    undetermined = "undetermined"
+
+
+class Residence(db.Model):
+    __tablename__ = "residences"
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
```

Comment:
I know the ID is typically `Initials+Room#` But would it be smart to have this as additional fields in the database? Would that be needed? Maybe this is a question for product/design.